docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+services:
+  alfio:
+    image: alfio/alf.io:1.16
+    environment:
+      POSTGRES_PORT_5432_TCP_PORT: "5432"
+      POSTGRES_PORT_5432_TCP_ADDR: postgres
+      POSTGRES_ENV_POSTGRES_DB: alfio
+      POSTGRES_ENV_POSTGRES_USERNAME: alfio
+      POSTGRES_ENV_POSTGRES_PASSWORD: alfio
+      SPRING_PROFILES_ACTIVE: dev
```

Comment:
Is this recommended on production too, or is session saved some other way then?

Reply:
if you don't activate it, sessions will be lost after a server restart. This means that you'll have to login again on the admin. The impact on a pending reservation is negligible.

It can be really useful in case you want to scale horizontally the number of alf.io instances, because it allows you to do it without relying on session stickiness.
With the jdbc-session profile active, you can have a load balancer in front of your n instances picking up the backend randomly. That's what we do on our production deployments.

If you've a single-server deployment, then you can choose not to activate it.